CMakeLists.txt

```diff
@@ -8,17 +8,25 @@ PROJECT(HPXMP CXX ASM)
 cmake_minimum_required(VERSION 2.8)
 
 set(hpxmp_sources
-    asm_functions.s
-    intel_hpxMP.cpp
-    hpx_runtime.cpp
-    kmp_atomic.cpp
-    loop_schedule.cpp)
+        src/asm_functions.s
+        src/intel_hpxMP.cpp
+        src/hpx_runtime.cpp
+        src/kmp_atomic.cpp
+        src/loop_schedule.cpp
+        src/ompt_hpx-general.cpp)
 
 set(hpxmp_headers
-    intel_hpxMP.h
-    hpx_runtime.h
-    kmp_atomic.h
-    loop_schedule.h)
+        src/intel_hpxMP.h
+        src/hpx_runtime.h
+        src/kmp_atomic.h
+        src/loop_schedule.h
+        src/ompt.h
+        src/ompt-event-specific.h
+        src/ompt-internal.h)
+
+if(HPXMP_WITH_OMPT)
```

Comment:
There should be a cmake cache variable defined for this, see for instance: https://github.com/STEllAR-GROUP/hpxMP/pull/8/files#diff-af3b638bc2a3e6c650974192a53c7291R23

Reply:
@hkaiser  Done, Thanks!